app/views/ReportView/components/GenomicSummary/components/KeyAlterations/index.scss

```diff
@@ -0,0 +1,25 @@
+// Shared styling between both versions
+.genomic-summary, .genomic-summary--print {
```

Comment:
this violates BEM pretty sure, `genomic-summary--print__alterations` should be `genomic-summary__alterations--print`

Reply:
sweet let me change it